app/src/main/java/chat/rocket/android/chatroom/adapter/ChatRoomAdapter.kt

```diff
@@ -204,6 +218,42 @@ class ChatRoomAdapter(
         }
     }
 
+    private val actionAttachmentOnClickListener = object : ActionAttachmentOnClickListener {
+        override fun onActionClicked(action: Action) {
+            val temp = action as ButtonAction
+            if (temp.url != null && temp.isWebView != null) {
+                if (temp.isWebView!!) {
+                    //Open in a configurable sizable webview
+                    Toast.makeText(context, "Open in a configurable sizable webview", Toast.LENGTH_SHORT).show()
+                } else {
+                    //Open in chrome custom tab
+                    with(this) {
+                        val customTabsBuilder = CustomTabsIntent.Builder()
```

Comment:
we have a `View.openTabbedUrl()` extension function on `chat.rocket.android.util.extensions.View.kt`